README.md

```diff
@@ -163,6 +163,16 @@ dir: /tmp/test
 resource: http://localhost:8080/rest/1
 ```
 
+Namespaces
+----------
+Currently, if the first use of a particular namespace occurs in RDF that is POSTed or PUT to the repository, regardless of any specific prefix binding supplied in the submitted graph, Fedora will instead bind the new namespace to a system-generated prefix in the form "ns00x".  While this behavior is not incorrect, it is inconvenient: prefixes generated during import will not match prefix bindings in the source repository. In order to avoid this behavior, follow the steps below:
+- Create a repository.json file and namespaces.cnd file, as described in [Best Practices - RDF Namespaces](https://wiki.duraspace.org/display/FEDORA4x/Best+Practices+-+RDF+Namespaces).
+- Start the destination repository for import with -Dfcrepo.modeshape.configuration=file:/path/to/repository.json, as described in [Application Configuration](https://wiki.duraspace.org/display/FEDORA4x/Application+Configuration).
+
+Note:
+- Setting -Dfcrepo.modeshape.configuration=file:/path/to/repository.json does not work if deploying Fedora using the one-click jar.
+- If restoring Fedora from a backup, prefix bindings in the backup will take precedence over bindings in namespaces.cnd.
```

Comment:
This statement is a little ambiguous. 

I agree that it is true when using /fcr:backup, but since this README is in the context of import/export, I can see potential confusion in interpreting this comment to relate to "import" from this utility.